instrumentation/nethttp/patch.go

```diff
@@ -12,3 +12,12 @@ func PatchHttpDefaultClient() {
 		http.DefaultClient = &http.Client{Transport: &Transport{RoundTripper: http.DefaultTransport}}
 	})
 }
+
+func PatchHttpDefaultClientWithPayload() {
```

Comment:
We should just update `PatchHttpDefaultClient` with options, instead of creating another one.

Reply:
done